test/pages/race/race_page_test.dart

```diff
@@ -0,0 +1,48 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:regatta_buddy/pages/race/race_page.dart';
+
+void main() {
+
+  testWidgets('Statistics are visible', (tester) async {
+    await tester.pumpWidget(const MaterialApp(home: RacePage()));
+
+    expect(find.text('Distance'), findsOneWidget);
+    expect(find.text('Time'), findsOneWidget);
+    expect(find.text('Avg. Speed'), findsOneWidget);
+  });
+
+  testWidgets('Can open actions modal', (tester) async {
+    await tester.pumpWidget(const MaterialApp(home: RacePage()));
+
+    expect(find.byIcon(Icons.warning_amber_rounded), findsOneWidget);
+
+    await tester.tap(find.byIcon(Icons.warning_amber_rounded));
+    await tester.pumpAndSettle();
+
+    expect(find.text('Needs help'), findsOneWidget);
+    expect(find.text('Protest'), findsOneWidget);
+    expect(find.text('Problem'), findsOneWidget);
+    expect(find.text('Cancel'), findsOneWidget);
+  });
+
+  testWidgets('Clicking action triggers notification which disappears after some time', (tester) async {
```

Comment:
To be super precise with clean tests practices this test should be divided into two, but for me it's good enough. Great PR